blogposts/announcing-sourcegraph-3.15.md

```diff
@@ -0,0 +1,296 @@
+
+Enable this feature by adding `"permissions.backgroundSync": {"enabled": true}` in your site configuration settings.
+
+## Experimental AND/OR operators for search contents
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://www.youtube.com/embed/ju0XEFU31W0" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://youtu.be/ju0XEFU31W0" target="_blank">View on YouTube</a></p>
+</p>
+-->
+
+Search for file contents using and- and or-expressions in queries. These operators enable more expressive searches in regexp and structural search modes. [The documentation](https://docs.sourcegraph.com/user/search/queries#operators) includes more details about how to these operators and their precedence and grouping.
+
+For example, if you want to find all of the places where two functions are within the same file, you can use the AND operator:
+
+EXAMPLE
+
+Or, if you want to find everywhere two functions are used, whether or not the other is present, you would use the OR operator:
+
+EXAMPLE
+
+You can even use them together:
+
+EXAMPLE
+
+Currently, operators only support searching file contents and do not apply to search scopes, you will see an alert if your query is not supported. 
+
+Enable this feature by adding `{"experimentalFeatures": {"andOrQuery": "enabled"}}` to your global, organization, or user settings.
+
+## Observability
+
+We continue to invest heavily in making Sourcegraph easier to monitor and debug out-of-the-box.
+
+### Monitoring
+
+- **Sourcegraph now monitors and has pre-defined alerts for 93 metrics across 14 services.** For now, these new alerts are warning-level alerts and in the future once more confidence in their accuracy has been gained they will be elevated to critical-level alerts.
+- 8 bug fixes to alerting and dashboards have been made
+- 11 changes to improve the legibility and visibility of dashboards have been made
+
+Additionally:
+
+- All dashboard panels now show an orange/red background when a metric is reaching its warning/critical threshold, making it easy to see which metrics are most interesting.
+- Search error rate and latency is now broken down by type on the Frontend dashboard:
+  - Search at a glance (browser web UI)
+  - Search-based code intelligence at a glance
+  - Search API usage at a glance
+
+**IMPORTANT:** If you have previously configured alert thresholds on the Frontend dashboard manually (instead of just on the Grafana Home dashboard as is typically reccomended), you will need to reconfigure those alert thresholds again after upgrading.
```

Comment:
```suggestion
**IMPORTANT:** If you have previously configured alert thresholds on the frontend dashboard manually (instead of on the Grafana home dashboard as typically recommended), you will need to reconfigure those alert thresholds again after upgrading.
```